snort-ip-blocklist-feed/info.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "snort-ip-blocklist-feed",
+  "version": "1.0.0",
+  "label": "Snort IP Blocklist Feed",
+  "category": "Threat Intelligence",
+  "description": "Snort is an open-source, free and lightweight network intrusion detection system (NIDS) software for Linux and Windows to detect emerging threats.",
```

Comment:
Please add the Threat Intel description after this description completed.
Ref: https://github.com/fortinet-fortisoar/connector-bambenek-feed/blob/develop/bambenek-feed/info.json#L4